apis/nucleus/src/components/ActionsToolbar.jsx

```diff
@@ -45,9 +45,9 @@ const ActionToolbarElement = {
 };
 
 const ActionsGroup = React.forwardRef(
-  ({ ariaExpanded = false, actions = [], first = false, last = false, addAnchor = false }, ref) =>
+  ({ ariaExpanded = false, actions = [], first = false, last = false, addAnchor = false, isRtl = false }, ref) =>
```

Comment:
This using isRtl as a property here.

Reply:
Have forwarded direction in the main element ActionsToolbar, calculated isRTL and using it inside the inner element, seems redundant to pass direction again and recalculate isRtl